pytorch_lightning/trainer/trainer.py

```diff
@@ -473,8 +474,6 @@ def fit(
         self.training_type_plugin.pre_training()
         self.precision_plugin.pre_training()
 
-        self.call_setup_hook(self.lightning_module)
```

Comment:
Please, move `on_before_accelerator_backend_setup ` before `self.accelerator_backend.setup`. It is a bug.